### ISSUE TITLE
Specific version package detail page not showing

### ISSUE THREAD

**Issue by devilmark84** [verdaccio/verdaccio#3006]
**Your Environment**
  
 * **verdaccio version**: 5.6.0
 * **node version** [12.x.x, 14.x.x]:
 * **package manager**: [npm@7, yarn@1]
 * **os**: [linux]
 * **platform**: [npm, docker]

**Describe the bug**

Can't visit a specific version page of a published package.

**To Reproduce**

I've just installed a brand new Verdiaccio based on Docker image. I tried publishing my first package and I made ```npm publish``` so everything went fine. Visiting the package page on my own Verdaccio instance (e.g. https://npm.iamdev.it/-/web/detail/vue-vuelidate-form), I can see the version I've just published.
In the "versions" tab, while I click on the version I've just published, I get "Sorry, we couldn't find it...".
https://npm.iamdev.it/-/web/detail/vue-vuelidate-form/v/0.2.0

**Expected behavior**

The specific version detail page of the package.

**Screenshots, server logs, package manager log**

**Configuration File (cat ~/.config/verdaccio/config.yaml)**
```lang=yml
# path to a directory with all packages
storage: /verdaccio/storage
# path to a directory with plugins to include
plugins: /verdaccio/plugins

web:
  enable: true
  title: IAMdev Private NPM Registry
  logo: /verdaccio/conf/logo.png
  primary_color: "#00aba8"
  gravatar: true
  sort_packages: desc
  darkMode: false
  favicon: /verdaccio/conf/favicon.png
  rateLimit:
    windowMs: 50000
    max: 1000
  pkgManagers:
    - npm
    - yarn
  login: true   

auth:
  ldap:
    type: ldap
    ...

uplinks:
  npmjs:
    url: https://registry.npmjs.org/
  yarn:
    url: https://registry.yarnpkg.com

packages:
  '@iamdev/*':
    access: $all
    publish: $authenticated
  '**':
    access: $all
    publish: $authenticated
    proxy: npmjs
```

**Environment information**

**Contribute to Verdaccio**

- [ ] I'm willing to fix this bug 🥇

**Comment by juanpicado** [comment]
Ok' I've noticed the same recently, l'll check soon.

**Comment by juanpicado** [comment]
fixed v5.6.1
